<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SN</class>
 <widget class="QWidget" name="SN">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SN</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgba(238, 255, 250, 200);</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>260</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>通过</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>110</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>写入SN/Mac</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>10</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	image: url(:/ProductTest_360/Tip.jpg);
border-style: outset;
border-width: 4px;
border-radius: 10px;
border-color: rgba(255, 170, 0, 30);
}
QLabel:hover{
background-color:rgba(100,255,100, 100);
border-color: rgba(255, 170, 0, 255);
color:rgba(0, 0, 0, 255);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>260</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>不通过</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ProductTest_360.qrc"/>
 </resources>
 <connections/>
</ui>
